### PR TITLE
fixing add user issue

### PR DESCRIPTION
**Pull request by willdoran**
[Review on Reviewable](https://reviewable.io/reviews/ushahidi/platform-client/46)

### CODE REVIEW

**Comment by jasonmule** [comment]
Looks good.

---







_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/ushahidi/platform-client/46#-:-K3_wRgCFKbr7M3x-LiW:-1921560736)_

**Comment by willdoran** [comment]
Fixed add user issue

File: app/setting/controllers/setting-users-controller.js [hunk 1]
```diff
@@ -64,6 +66,10 @@ function (
         return false;
     };
 
+    $scope.addUser = function () {
+        $location.path('/settings/users/create');
```
Comment (rjmackay) [inline app/setting/controllers/setting-users-controller.js]: We should really never be doing this. This is clearly a link, so use a link not a button and let browser navigation take car of it. If the CSS doesn't work as a button, thats a bug in the PL.

Comment (willdoran) [inline app/setting/controllers/setting-users-controller.js]: I'll add it as an issue under the PL.

Comment (rjmackay) [inline app/setting/controllers/setting-users-controller.js]: I *think* it actually already works. Using `class="button button-secondary"` on an `<a>` get the right visual

**Pull request merged**